packages/api/src/libs/queries/emp.ts

```diff
@@ -192,17 +158,20 @@ export default (appState: Dependencies) => {
     const addresses = Array.from(appState.registeredEmps.values());
     return sumTvm(addresses, currency);
   }
-  async function getTotalTvl(currency: CurrencySymbol = "usd") {
+  async function getTotalTvlSample(currency: CurrencySymbol = "usd") {
     assert(appState.stats.emp[currency], "Invalid currency: " + currency);
-    const { value } = await appState.stats.emp[currency].latest.tvl.getGlobal();
-    return value;
+    return appState.stats.emp[currency].latest.tvl.getGlobal();
+  }
+  async function getTotalTvl(currency: CurrencySymbol = "usd") {
+    const result = await getTotalTvlSample(currency);
+    return result.value;
   }
 
   return {
-    getFullEmpState,
-    getAnyEmp,
-    listActiveEmps,
-    listExpiredEmps,
+    getFullState,
+    getAny,
+    listActive,
+    listExpired,
```

Comment:
these are renamed to be the same across queries for emp and lsp so i can loop through easily